symbolic-symcache/src/format.rs

```diff
@@ -209,11 +209,18 @@ impl FuncRecord {
     }
 
     /// The instruction address _after_ the end of the function.
+    ///
+    /// TODO(ja): Describe special case.
```

Comment:
The special case is: len of 0xffff means we don't know how big it is and therefore assume it goes all the way to the end.